### PR TITLE
Install Zts-Xcrypt using Governance Policies

### PR DESCRIPTION
**Pull request by mahesh-zetta**
Governance Policy is used to configure RBAC on the managed clusters and install Zettaset encryption.

Signed-off-by: Mahesh Bhoothapuri <mahesh@zettaset.com>

### CODE REVIEW

File: stable/CM-Configuration-Management/README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Policy  | Description | Prerequisites
 [policy-namespace](../CM-Configuration-Management/policy-namespace.yaml) | Ensures that a namespace exists as specified. |
 [policy-pod](../CM-Configuration-Management/policy-pod.yaml) | Ensures that a pod exists as specified. |
 [policy-zts-cmc](../CM-Configuration-Management/policy-zts-cmc.yaml) | This example deploys a replica of \`zts-cmc-deployment\`. | See the [Zettaset README.stable(https://github.com/zettaset/zettaset-public/) to learn more about Zettaset CMC Deployment.
+[policy-zts-xcrypt-rbac](../CM-Configuration-Management/policy-zts-xcrypt-rbac.yaml) | This example deploys RBAC for \`zts-xcrypt-deployment\`. | See the [Zettaset README.stable(https://github.com/zettaset/zettaset-public/) to learn more about Zettaset Xcrypt Deployment.
```
Comment (gparvin) [inline stable/CM-Configuration-Management/README.md:12]: Can we have these contributions initially in the community folder?  We always want contributions to begin as a community donation and decide later if that contribution can be moved to stable.  

The link https://github.com/zettaset/zettaset-public/ does not seem to provide any details on ZettaSet or how to receive support for this policy either.

**Review by gparvin — COMMENTED** [review]
Before we add more ZettaSet policies to stable I think we need some more clarity on how customers will get support for the existing policy using the link provided: https://github.com/zettaset/zettaset-public/
For now lets put these new policies in the community folder.  Thanks

**Comment by gparvin** [comment]
I think this was completed in a different pr.  Re-open if I'm wrong.

**Comment by mahesh-zetta** [comment]
The Xcrypt policy was commited as a PolicySet in the Community branch.
https://github.com/stolostron/policy-collection/tree/main/policygenerator/policy-sets/community/zts-xcrypt

**Pull request closed without merging**